assignment

```diff
@@ -0,0 +1,87 @@
+1.
```

Comment:
This code should be in your fork of the GT-IDEaS/SkillsWorkshop2017 repository, not the GT-IDEaS/planets repository.  Also, each problem should be in its own file.  See the README file [here](https://github.com/GT-IDEaS/SkillsWorkshop2017/tree/master/Week01#week-01-introduction-to-python) for organization details.